jambi/config.py

```diff
@@ -0,0 +1,26 @@
+import os
+
+from jambi.exceptions import ImproperlyConfigured
+
+ENVIRONMENT_VARIABLE = "JAMBI_CONFIG"
+DEFAULT_CONFIG_FILENAME = "jambi.conf"
+
+def get_config_file():
+	config_file = os.environ.get(ENVIRONMENT_VARIABLE)
+	if config_file:
+		if os.path.isfile(config_file):
+			return config_file
+		else:
+			raise ImproperlyConfigured(
+				"{} environment variable is set, but no file was found" \
+				.format(ENVIRONMENT_VARIABLE)
+			)
+	else:
+		if os.path.isfile(DEFAULT_CONFIG_FILENAME):
+			return DEFAULT_CONFIG_FILENAME
+		else:
+			raise ImproperlyConfigured(
+				"No configuration file found. Either create {} in the " \
+				"install directory, or set the {} environment variable." \
```

Comment:
Potentially confusing to people who might not know what the 'install directory' is. Maybe alternative wording would be something like. "[...]Either create jambi.conf in _this_ directory[...]"

What do you think?

Edit: In fact, we could just remove the bit "in the install directory"